tools/flang2/flang2exe/ili.h

```diff
@@ -162,10 +162,12 @@ typedef enum ILIO_KIND {
   ILIO_512LNK = 32,
   ILIO_X87LNK = 33,
   ILIO_DOUBLEDOUBLELNK = 34,
-  ILIO_FLOAT128LNK = 35
+  ILIO_FLOAT128LNK = 35,
+  /* just for debug to dump ili */
+  ILIO_QP = 36
 } ILIO_KIND;
 
-#define ILIO_MAX 35
+#define ILIO_MAX 36
 #define ILIO_ISLINK(n) ((n) >= ILIO_IRLNK)
```

Comment:
IMHO, Here should add a special case for `ILIO_QP` that does not belong to `LINK`, 
such as `#define ILIO_ISLINK(n) ((n) >= ILIO_IRLNK && (n) != ILIO_QP)`.
And at line 289, the macro `IL_ISLINK` also needs to add a special case for `ILIO_QP`.

Reply:
Great! Thanks for your reply, I think an alternative modification method is putting `ILIO_QP` after `ILIO_FLOAT128`, is that ok?

Reply:
Yes, it is also effective and even better.

Reply:
> Great! Thanks for your reply, I think an alternative modification method is putting `ILIO_QP` after `ILIO_FLOAT128`, is that ok?

Yes, it is also effective and even better.